### PR TITLE
13987 invalid Court Order component should prevent filing

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#13987

*Description of changes:*
  - app version = 3.8.12
  - removed unnecessary setters
  - added $nexttick after setting app validate to allow validation to happen

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -759,8 +759,8 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // Prompt app validations.
     this.setAppValidate(true)
 
-    // Evaluate valid flags. Scroll to invalid components or file alteration.
-    if (await this.validateAndScroll(this.getFlagsReviewCertify, ComponentsReviewCertify)) {
+    // Evaluate valid flags. Scroll to invalid components or file alteration. Wait to allow app validation.
+    if (await this.$nextTick.validateAndScroll(this.getFlagsReviewCertify, ComponentsReviewCertify)) {
```
Comment (lambert-alex) [inline src/App.vue:763]: The validations checks were happening before court order could be set invalid.

File: src/views/Change.vue [hunk 2]
```diff
@@ -129,7 +129,6 @@ export default class Change extends Mixins(
   @Action setHaveUnsavedChanges!: ActionBindingIF
   @Action setFilingId!: ActionBindingIF
   @Action setDocumentOptionalEmailValidity!: ActionBindingIF
-  @Action setValidCourtOrder!: ActionBindingIF
```
Comment (lambert-alex) [inline src/views/Change.vue:132]: Not used

File: src/views/Conversion.vue [hunk 3]
```diff
@@ -83,7 +83,6 @@ export default class Conversion extends Mixins(
   // Global actions
   @Action setHaveUnsavedChanges!: ActionBindingIF
   @Action setFilingId!: ActionBindingIF
-  @Action setValidCourtOrder!: ActionBindingIF
```
Comment (lambert-alex) [inline src/views/Conversion.vue:86]: Not used

File: src/views/SpecialResolution.vue [hunk 4]
```diff
@@ -174,11 +174,6 @@ export default class SpecialResolution extends Mixins(
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
-  /** Check validity state, only when prompted by app. */
-  get invalidCourtOrder (): boolean {
```
Comment (lambert-alex) [inline src/views/SpecialResolution.vue:178]: Not used

**Comment by lambert-alex** [comment]
I have a git push that is stuck... It fixes the lint and adds the package version. I'll push it from home.

File: src/App.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/App.vue:763]: Please make this 2 separate statements. (Maybe move `await Vue.nextTick()` right after setAppValidate above.)

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**